### PR TITLE
add gitlab-ci yaml

### PR DESCRIPTION
**Pull request by General-Beck**
Build script for Gitlab CI
[ci-skip]

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    - SET
+    - rustup default stable-x86_64-pc-windows-msvc
+    - cargo build --release --verbose
+  tags:
+   - rust-windows
+  artifacts:
+    paths:
+    - target/release/parity.exe
+    - target/release/parity.pdb
+    - target/release/parity.lib
```
Comment (arkpar) [inline .gitlab-ci.yml]: .lib and .exp are not needed

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    - rust-beta
+  artifacts:
+    paths:
+    - target/release/parity
+    name: "${CI_BUILD_NAME}_parity"
+linux-stable:
+  stage: build
+  image: ethcore/rust:stable
+  script:
+    - export CCX="clang"
```
Comment (rphmeier) [inline .gitlab-ci.yml]: CCX?

Comment (General-Beck) [inline .gitlab-ci.yml]: @rphmeier for rocksdb

Comment (rphmeier) [inline .gitlab-ci.yml]: it's `CXX` some places and `CCX` in some

Comment (General-Beck) [inline .gitlab-ci.yml]: @rphmeier ah, ok!

**Pull request merged**